src/tyro/__init__.py

```diff
@@ -11,3 +11,7 @@
 # Deprecated interface.
 if not TYPE_CHECKING:
     from ._deprecated import *  # noqa
+
+
+# TODO: this should be synchronized automatically with the pyproject.toml.
+__version__ = "0.7.1"
```

Comment:
@brentyi  tankuuuuuu!!

i have only ever done version sync thru setup.py but allegedly this works for pyproject.toml thingies  https://stackoverflow.com/a/67097076

Reply:
Maybe something wrong with environment, but I just tried this and unfortunately it seemed flakey for me... kept returning `0.6.0` after I installed `0.7.1`.